packages/create-vite/LICENSE

```diff
@@ -23,6 +23,127 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
+# License of the files in the directories starting with "template-" in create-vite
+The files in the directories starting with "template-" in create-vite and files
+generated from those files are licensed under the CC0 1.0 Universal license:
```

Comment:
I replaced "files generated by create-vite" with "files generated from those files" so that we don't say anything about files generated by the redirected CLIs (e.g. create-vue, create svelte).